### ISSUE TITLE
Examples for how to install regular npm packages into a built widget

### ISSUE THREAD

**Issue by paddymul** [widgetti/ipyreact#35]
I see that there are unpkg recommendations.  What about regular npm packages.

**Comment by paddymul** [comment]
So in @kolibril13 's TLDraw package, I see 
`import { Tldraw } from "@tldraw/tldraw";` in  https://github.com/kolibril13/jupyter-tldraw/blob/main/tldraw/comp.tsx#L2

But I don't see where `@tldraw` is defined as a package, or tied to NPM in the repo.

**Comment by maartenbreddels** [comment]
We fetch it from https://esm.sh/ at runtime. I see this is not documented though. But I'd also like to support non-esm bundles, so we could just use webpack/vite/whatever to not depend on this cdn.

**Comment by kolibril13** [comment]
My thoughts on this:
* When I install jupyter-tldaw from pypi with `pip install tldraw`, it comes without the react component. Is there maybe a way to package the component to pypi itself? (I used poetry to publish)
* After initiating the jupyter-tldaw widget in jupyterlab for the first time, it is permanently installed. I disconnected from internet and restarted the kernel, restarted the computer, and the widget was in each case still offline available in JupyterLab. 
    * However, jupyter-tldaw was not available in VS Code notebook, although I used the same jupyter kernel. After reconnecting to the internet and first initiating in VS Code, it was also permanently available offline in VS Code.
 
That brings me to the question: Can there maybe a widget property that describes where the react code is saved on disk?
On that note, `get_state` looks currently like this:
<img width="800" alt="image" src="https://github.com/widgetti/ipyreact/assets/44469195/d3ccd44d-8d1f-4b9b-8efe-754d47c5e07d">
